src/unit_tests/syscheckd/whodata/test_win_whodata.c

```diff
@@ -78,7 +78,8 @@ int SIZE_EVENTS;
 
 const PWCHAR WCS_TEST_PATH = L"C:\\Windows\\a\\path";
 const char *STR_TEST_PATH = "c:\\windows\\a\\path";
-
+int __real_OSHash_Add(OSHash *hash, const char *key, void *data);
+void *__real_OSHash_Get(const OSHash *self, const char *key);
```

Comment:
Same as a previous comment on `__real_OSHash_Create`, we might want to move this declarations to `hash_op_wrappers.h`